.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  - package-ecosystem: gitsubmodule
+    commit-message:
+      prefix: "chore"
```

Comment:
We don't write `chore:` in CFEngine repos.
```suggestion
```

Reply:
Yea, let's remove it. We already have other things we want to put before colon (ticket number, or filename / component name, or `CI:`, `GH Actions:`, `Tests:` etc.) See:

https://github.com/cfengine/core/blob/master/CONTRIBUTING.md

Reply:
https://github.com/cfengine/buildscripts/pull/1214